config/api.yml

```diff
@@ -2073,6 +2073,8 @@
         :identifier: ems_infra_new
       - :name: edit
         :identifier: ems_infra_edit
+      - :name: refresh
+        :identifier: ems_cloud_refresh
```

Comment:
I've not seen multiple "names" in the list before, so I'm not sure what happens... Is there an example of this elsewhere in the file?  @abellotti ?

Reply:
@Fryguy Yeah exactly, I just put it there "for now" so it was easier for me tbh to comment out `ems_infra_refresh` during testing 😂